.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,4 @@
+<!-- Keep the progress checklist.  -->
 ## Progress
```

Comment:
I'm on the fence about having this progress section, I lean towards removing it. Curious what others think about removing it?